student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# Class Student inherited from Person
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
[OPTIONAL]
- [ ] It is not standard to pass keyword arguments to your `super()` as you are doing with parent_permission so i kindly advice that you make the necessary corrections using the code snippet below 
- code snippet
```ruby
 super(age, name, parent_permission)
```